component-versions.md

```diff
@@ -2,6 +2,61 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated this is called out in the [release notes](https://discourse.ubuntu.com/t/release-notes/17842) and components are marked with `n/a` below.
 
+[Details=1.16.1]
+
+### Charms
+
+n/a
```

Comment:
This is not correct. You will need the charm / bundle and 3rd party charm revisions from the deployment-scripts stable-1.16 build once it has finished